src/Element/NoteFooter.tsx

```diff
@@ -111,27 +112,65 @@ export default function NoteFooter(props: NoteFooterProps) {
     }
     return (
       <>
-        <div className={`reaction-pill ${hasReacted('+') ? 'reacted' : ''} `} onClick={(e) => react("+")}>
+        <div className={`reaction-pill ${hasReacted('+') ? 'reacted' : ''} `} onClick={() => react("+")}>
           <div className="reaction-pill-icon">
             <FontAwesomeIcon icon={faHeart} />
           </div>
           <div className="reaction-pill-number">
             {formatShort(groupReactions[Reaction.Positive])}
           </div>
         </div>
-        <div className={`reaction-pill ${hasReacted('-') ? 'reacted' : ''}`} onClick={(e) => react("-")}>
-          <div className="reaction-pill-icon">
-            <FontAwesomeIcon icon={faThumbsDown} />
-          </div>
-          <div className="reaction-pill-number">
-            {formatShort(groupReactions[Reaction.Negative])}
-          </div>
-        </div>
         {repostIcon()}
       </>
     )
   }
 
+  async function share() {
+    const url = `${window.location.protocol}//${window.location.host}/e/${hexToBech32("npub", ev.Id)}`;
+    if ("share" in window.navigator) {
+      await window.navigator.share({
+        title: "Snort",
+        url: url
+      });
+    } else {
+      await navigator.clipboard.writeText(url);
+    }
+  }
+
+  function menuItems() {
+    return (
```

Comment:
When we add note mention we could add another option here for copying the raw note id so you can paste it in a note for mentioning it.